src/codegen/common.ts

```diff
@@ -592,26 +590,27 @@ export const generateRuntime: CodegenHelpers.GenerateRuntimeFunction = (
       schemaId
     );
   };
-  
+
   const httpFetch = (operationName, options) => {
     const reqBody = options.body || null;
     const userHeaders = options.headers || {};
     const headers = {
       ...userHeaders,
       'Content-Type': 'application/json',
     };
-  
+
     const timeoutMs = 30_000;
-  
+
     const reqOptions = {
       method: 'POST',
       headers: headers,
       timeout: timeoutMs,
       body: reqBody,
     };
-  
-    const netlifyGraphUrl = makeNetlifyGraphUrl({ operationName: operationName });
-  
+
+    const siteId = options.siteId || process.env.SITE_ID;
+    const netlifyGraphUrl = makeNetlifyGraphUrl({ operationName: operationName, siteId });
```

Comment:
Mind picking a style? I prefer no-punning to make it work in all the known JS universes, so:
```suggestion
    const netlifyGraphUrl = makeNetlifyGraphUrl({ operationName: operationName, siteId: siteId });
```

Reply:
yup!